stablehlo/dialect/VhloAttrs.td

```diff
@@ -22,6 +22,25 @@ include "stablehlo/dialect/VhloEnums.td"
 
 include "mlir/IR/AttrTypeBase.td"
 
+//===----------------------------------------------------------------------===//
+// Attribute Versioning
+//===----------------------------------------------------------------------===//
+
+class VHLO_AttrDef<string name,
+                   string minVersion = "0.3.0",
+                   string maxVersion = "current">
+  : AttrDef<VHLO_Dialect, name, [VHLO_VersionedAttrInterface]> {
+  let extraClassDeclaration = [{
+    mlir::vhlo::Version getMinVersion() {
+      return *mlir::vhlo::Version::fromString("}] #  minVersion # [{");
```

Comment:
This is something that I missed when reviewing the previous pull request, but what happens if someone accidentally specifies a version that doesn't parse. Can we `llvm::report_fatal_error` instead of dereferencing an empty optional?

Reply:
Done. Should be fairly uncommon since this only applies to the hardcoded versions in TD files, but is a negligible performance hit for a better user experience. Generated code now reads: `"invalid version <version> in <(op|type|attr)-name>"`

```
  mlir::vhlo::Version getMinVersion() {
    auto version = mlir::vhlo::Version::fromString("0.3.0");
    if (failed(version)) llvm_unreachable("invalid version 0.3.0 in ArgResultAlias");
    return *version;
  }
```